answers.txt

```diff
@@ -0,0 +1,38 @@
+• What is a gitignore file and what should it include for a Java project?
+
+	A gitignore file is used to tell git to ignore certain files while tracking all the changes in our project directory.
+	These files won't be tracked by git and thus changes made to those files will be ignored.
+	While creating .gitignore file we can see that it includes file extensions for files which we do not want to track. 
+	
+	For a java project there are different files which we do not want to include in source control.
```

Comment:
Can you please specify what kind of files will be ignored in java for better understanding ?

Reply:
The gitignore file, as mentioned, includes compiled class files(.class), log files(.log), package files(.jar, .war, .zip), .project file, etc.